packages/amplication-server/prisma/migrations/20201006094009/README.md

```diff
@@ -0,0 +1,142 @@
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("buildId")REFERENCES "public"."Build"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("environmentId")REFERENCES "public"."Environment"("id") ON DELETE CASCADE ON UPDATE CASCADE
+
+ALTER TABLE "public"."Deployment" ADD FOREIGN KEY ("actionId")REFERENCES "public"."Action"("id") ON DELETE CASCADE ON UPDATE CASCADE
+```
+
+## Changes
+
+```diff
+diff --git schema.prisma schema.prisma
+migration 20201004110208..20201006094009
+--- datamodel.dml
++++ datamodel.dml
+@@ -1,7 +1,7 @@
+ datasource db {
+   provider = "postgresql"
+-  url = "***"
++  url = "***"
+ }
+ generator client {
+   provider        = "prisma-client-js"
+@@ -52,8 +52,9 @@
+   commits       Commit[]
+   lockedEntitis Entity[]
+   builds        Build[]
++  deployments   Deployment[]
+   @@unique([accountId, organizationId])
+ }
+ model UserRole {
+@@ -81,8 +82,9 @@
+   roles          AppRole[]
+   commits        Commit[]
+   builds         Build[]
++  environments Environment[]
+   @@unique([organizationId, name])
+ }
+ model AppRole {
+@@ -302,12 +304,13 @@
+   Success
+ }
+ model Action {
+-  id        String       @id @default(cuid())
+-  createdAt DateTime     @default(now())
+-  steps     ActionStep[]
+-  builds    Build[]
++  id          String       @id @default(cuid())
++  createdAt   DateTime     @default(now())
++  steps       ActionStep[]
++  builds      Build[]
++  deployments Deployment[]
+ }
+ model ActionStep {
+   id          String           @id @default(cuid())
+@@ -361,6 +364,47 @@
+   action         Action          @relation(fields: [actionId], references: [id])
+   actionId       String
++  deployments Deployment[]
+   @@unique([appId, version])
+ }
++
++model Environment {
++  id          String   @id @default(cuid())
++  createdAt   DateTime @default(now())
++  updatedAt   DateTime @default(now())
++  app         App      @relation(fields: [appId], references: [id])
++  appId       String
++  name        String
++  description String?
++  address     String
++
++  deployments Deployment[]
++  @@unique([appId, name])
++}
++
++enum EnumDeploymentStatus {
++  Completed
++  Waiting
++  Active
++  Delayed
++  Failed
++  Paused
++}
++
++model Deployment {
++  id            String               @id @default(cuid())
++  createdAt     DateTime             @default(now())
++  createdBy     User                 @relation(fields: [userId], references: [id])
++  userId        String
++  build         Build                @relation(fields: [buildId], references: [id])
++  buildId       String
++  environment   Environment          @relation(fields: [environmentId], references: [id])
++  environmentId String
++  status        EnumDeploymentStatus
++  message       String?
++  action        Action               @relation(fields: [actionId], references: [id])
++  actionId      String
++
++
```

Comment:
Unneeded extra space

Reply:
Did you notice that you are review the auto-generated .md file ?

Reply:
No sorry